packages/cpp-debug/src/browser/editable-widget/memory-editable-table-widget.tsx

```diff
@@ -185,23 +184,26 @@ export class MemoryEditableTableWidget extends MemoryTableWidget {
     }
 
     protected submitMemoryEdits = async (): Promise<void> => {
-        // eslint-disable-next-line @typescript-eslint/no-unused-vars
-        for await (const _ of this.submitMemoryEditInOrder()) {
-            // Do nothing, but without lint errors.
+        for (const edit of this.submitMemoryEditInOrder()) {
+            try {
+                await this.memoryProvider.writeMemory(edit);
+            } catch (e) {
+                console.log('Problem writing memory with arguments', edit, '\n', e);
+            }
         }
     };
 
-    private * submitMemoryEditInOrder(): IterableIterator<Promise<DebugProtocol.WriteMemoryResponse | undefined>> {
+    private * submitMemoryEditInOrder(): IterableIterator<DebugProtocol.WriteMemoryArguments> {
```

Comment:
Do you think it might make sense to rename this to something else since the the memory edits are not actually submitted here? Also maybe we can consider moving the two `this.memoryEditsCompleted` operations to the parent function (`submitMemoryEdits`) and make this function return an array instead of a generator to simplify things.

Reply:
Sounds like a good idea. I wrote it this way previously to (overly elaborately) work around a lint rule in our repository, and it isn't really necessary.